.travis.yml

```
@@ -1,24 +1,38 @@
 language: dart
 
 dart:
-  - 2.7.0
   - dev
 
-dart_task:
-  - test: --platform vm
-    xvfb: false
-  - test: --platform firefox
-  - dartanalyzer: --fatal-infos --fatal-warnings .
-
-matrix:
+jobs:
   include:
-  # Only validate formatting using the dev release
-  - dart: dev
-    dart_task: dartfmt
+    - stage: analyze_and_format
+      name: "Analyze lib/"
+      os: linux
+      script: dartanalyzer --enable-experiment=non-nullable --fatal-warnings --fatal-infos lib/
+    - stage: analyze_and_format
+      name: "Analyze test/"
+      os: linux
+      script: dartanalyzer --enable-experiment=non-nullable --fatal-warnings --fatal-infos test/
```

```suggestion
      script: dartanalyzer --fatal-warnings --fatal-infos test/
```

Or you could also consider combining these again. https://github.com/dart-lang/csslib/blob/452a2e5b42bca5d90d7dfdba51555cbeead4fd56/.travis.yml#L8-L11